### DISCUSSION TITLE
能否支持播放玩是否保留最后一帧的配置

### DISCUSSION THREAD

**Discussion by wu781521512** [Tencent/libpag#480] [General] [answered]
我在Android上测试，有些需要保留最后一帧，有写不需要
我看Demo是会保留最后一帧
我现在不保留，是播放完成后调用
pagView.setComposition(null);
pagView.flush();
效果上是播放完成就消失了
但是不确定是否合理

**Comment by domchen** [comment, marked as answer]
播放一遍都是停留在最后一帧的，如果想回到第一帧设置setProgress(0)在flush就好，如果只是暂时想清空画面，对整个root Composition设置visible=false再flush会更好。直接清空Composition下次如果再设置回来，会导致内部缓存都丢失，是个比较重的操作。如果你确定后面再也不需要第二次播放这个素材，那这样操作是对的。
